src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/PaymentOrder/Update/UpdateTests.cs

```diff
@@ -1,41 +1,42 @@
-﻿using Atata;
-using NUnit.Framework;
-using Sample.AspNetCore.SystemTests.PageObjectModels.Base;
-using Sample.AspNetCore.SystemTests.Services;
-using Sample.AspNetCore.SystemTests.Test.Helpers;
-using SwedbankPay.Sdk;
-using System;
-using System.Linq;
-using System.Threading.Tasks;
-
-namespace Sample.AspNetCore.SystemTests.Test.PaymentTests.PaymentOrder.Update
-{
-    public class UpdateTests : Base.PaymentTests
-    {
-        public UpdateTests(string driverAlias)
-            : base(driverAlias)
-        {
-        }
-
-
-        [Test]
-        [TestCaseSource(nameof(TestData), new object[] { false, PaymentMethods.Card })]
-        public async Task Update_PaymentOrder(Product[] products, PayexInfo payexInfo)
-        {
-            GoToPayexSwishPaymentFrame(products.Skip(1).ToArray())
-                .Pay.Content.Should.BeEquivalent($"Betala {string.Format("{0:N2}", Convert.ToDecimal(products.Skip(1).First().UnitPrice / 100 * products.Skip(1).First().Quantity))} kr")
-                .SwitchToRoot<HomePage>()
-                .Header.Products.ClickAndGo();
-
-            GoToOrdersPage(products, payexInfo, Checkout.Option.Anonymous)
-                .PaymentOrderLink.StoreValue(out var orderLink)
-                .Actions.Rows[y => y.Name.Value.Contains(PaymentOrderResourceOperations.CreatePaymentOrderCancel)].Should.BeVisible()
-                .Actions.Rows[y => y.Name.Value.Contains(PaymentOrderResourceOperations.CreatePaymentOrderCapture)].Should.BeVisible()
-                .Actions.Rows[y => y.Name.Value.Contains(PaymentOrderResourceOperations.PaidPaymentOrder)].Should.BeVisible()
-                .Actions.Rows.Count.Should.Equal(3);
-
-            var order = await SwedbankPayClient.PaymentOrders.Get(orderLink, SwedbankPay.Sdk.PaymentOrders.PaymentOrderExpand.All);
-
-        }
-    }
+﻿using Atata;
+using NUnit.Framework;
+using Sample.AspNetCore.SystemTests.PageObjectModels.Base;
+using Sample.AspNetCore.SystemTests.Services;
+using Sample.AspNetCore.SystemTests.Test.Helpers;
+using SwedbankPay.Sdk;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Sample.AspNetCore.SystemTests.Test.PaymentTests.PaymentOrder.Update
+{
+    public class UpdateTests : Base.PaymentTests
+    {
+        public UpdateTests(string driverAlias)
+            : base()

```

Comment:
Codacy found an issue: [Remove this redundant 'base()' call.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5523315)